tests/fv3_conf/control_run.IN

```diff
@@ -67,10 +69,17 @@ else
 
 fi
 
-cp    @[INPUTDATA_ROOT]/${inputdir}/INPUT/aerosol.dat .
-cp    @[INPUTDATA_ROOT]/${inputdir}/INPUT/co2historicaldata_201*.txt .
-cp    @[INPUTDATA_ROOT]/${inputdir}/INPUT/sfc_emissivity_idx.txt .
-cp    @[INPUTDATA_ROOT]/${inputdir}/INPUT/solarconstant_noaa_an.txt .
+if [ $NPX = 49 ]; then
```

Comment:
Are these input files (`.txt` files) different for low resolution data? I didn't know they are resolution-dependent. This seems confusing to me.

Or is it just that the directory structure different for `FV3_input_data48`?

Reply:
Those files are the same. But in current C96 and other tests, those fixed files are not in inputdir directory like other fixed files, they are put in INPUT directory where the ICs stay. Since we plan to get rid of this INPUT directory, I was thinking to clean it up when all tests start to use INPUT_L127 tests.

Reply:
Sounds reasonable, thanks!